TestProjects/fbx-api-tests/Packages/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "dependencies": {
```

Comment:
Thanks Benoit for reviewing. This api scraping test needs to be **run on only one editor**. Because for all editors, the apis of the package are usually the same. So we often run the api scaring test on the lowest editor version.
But FBX master branch is special. I find the APIs are different between using `com.unity.recorder < 4.0.0` and using `com.unity.recorder >= 4.0.0`. Plus `Unity.Formats.Fbx.Editor.api` file in the package was generated with `"com.unity.recorder": "4.0.0"`.
So I have to run this test on 2022.1 which supports `"com.unity.recorder": "4.0.0"`.
As for keeping it up to date, the version in `manifest.json` is a minimum version. After we release a new FBX version, editor will use the latest package version automatically.